Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -113,7 +118,7 @@ if [ "$ARM_PKG_CONFIG_NOT_PRESENT" -eq 1 ]; then
     $APT_GET install pkg-config
     if [ -f /usr/share/pkg-config-crosswrapper ]; then
         # We are on non-Ubuntu so simulate effect of installing pkg-config-arm-linux-gnueabihf.
```

Comment:
> changing to ln -sf is to allow the script to be run twice (if first run fails)

The comment right above this change says we are on non-Ubuntu, so *something* is wrong here.

If the comment is right we shouldn't be in here when installing jammy

If the comment is wrong we should remove it.

My guess is the former.

Reply:
Or Ubuntu change things to be closer to debian again...